<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanFittingWindow</class>
 <widget class="QMainWindow" name="PanFittingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fitting Result</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBoxPan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>530</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitting for Pan-Genome Profile Curve</string>
    </property>
    <widget class="QLabel" name="labelPanModelA">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPModelName1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>120</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPModelName1_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>25</y>
       <width>200</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameters and Fitting Result:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPanModelA_Parameters">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>45</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPanModelA_AValue95Interval">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPanModelA_CValue95Interval">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>95</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPanModelA_RSquared">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPanModelADescription">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxCore">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>530</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Fitting for Core Genome Profile Curve</string>
    </property>
    <widget class="QLabel" name="labelCoreModelA">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>120</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Model:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPModelName1_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>25</y>
       <width>120</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Parameters:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCoreModelADescription">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCoreModelA_Parameters">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>45</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCoreModelA_AValue95Interval">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCoreModelA_CValue95Interval">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>95</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCoreModelA_RSquared">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>300</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
